client/src/reader/containers/ReaderFullNotes/index.js

```diff
@@ -99,10 +100,14 @@ function ReaderFullNotesContainer({
     return rgms?.length ? rgms : [];
   }
 
+  const { t } = useTranslation();
+  const myNotes = t("reader.menus.notes.my_notes"); // Set translations for getOverlayPropsForGroup(), cannot use hook inside a conditional statement
+  const allNotes = t("reader.menus.notes.all_notes");
+
   function getOverlayPropsForGroup() {
     return {
-      title: readingGroup === "me" ? "My Notes" : readingGroup.attributes.name,
-      subtitle: readingGroup === "me" ? null : "All notes for group:",
+      title: readingGroup === "me" ? myNotes : readingGroup.attributes.name,
+      subtitle: readingGroup === "me" ? null : allNotes,
```

Comment:
Right, so you can't call the hook conditionally, as in line 103 there. But you can use `t` conditionally once it's defined. I would just move lines 104-105 inside `getOverlayPropsForGroup`.